lib/trivial.nix

```diff
@@ -232,6 +232,23 @@ in {
 
   nixpkgsVersion = builtins.trace "`lib.nixpkgsVersion` is deprecated, use `lib.version` instead!" version;
 
+  /* The output path of a derivation to fetch this nixpkgs if available,
+     otherwise the store path to this nixpkgs. This can be used to incur a
+     dependency on this nixpkgs.
```

Comment:
This value is never the output path of a derivation.

Reply:
Oh, you're right, I didn't realize Nix doesn't store derivers for builtin fetchers, I thought it did for some reason.